### PR TITLE
add comments to batched_gemm

### PR DESCRIPTION
**Pull request by j4yan**
Add documentation to batched_gemm. Also renamed `ComputeBasePtr` to `ComputePtrOffset`.

### CODE REVIEW

File: include/ck/tensor_operation/gpu/device/device_batched_gemm_xdl.hpp [hunk 1]
```diff
@@ -16,6 +16,31 @@ namespace ck {
 namespace tensor_operation {
 namespace device {
 
+/*
+ * \brief Wrapper function of GridwiseGemm::Run to realize BatchedGEMM.
+ *
+ * \tparam ComputePrtOffsetOfBatch Class that computes the base pointer offsets of A, B, C matrix
```
Comment (aosewski) [inline include/ck/tensor_operation/gpu/device/device_batched_gemm_xdl.hpp:22]:
```suggestion
 * \tparam ComputePtrOffsetOfBatch Class that computes the base pointer offsets of A, B, C matrix
```

File: include/ck/tensor_operation/gpu/device/device_batched_gemm_xdl.hpp [hunk 2]
```diff
@@ -16,6 +16,31 @@ namespace ck {
...
+ * given the batch. For example, ComputePtrOffsetOfStridedBatch() computes the offsets of evenly
+ * strided batched, but we can easily extend to other layouts. The returned offset can be either \p
+ * index_t or \p long_index_t. If it returns \p long_index_t, we are not subject to the 2GB
+ * limitations.
+ *
+ * \tparam Block2CTileMap Block2CTileMap::CalculateBottomIndex() takes in id of a workgroup and
+ * returns the 2D index of the tile that it computes. \see
+ * GridwiseGemm_k0mk1_k0nk1_mn_xdlops_v2r3::Run().
+ *
+ * \note Using \p ComputePrtOffsetOfBatch gives us the flexibility that 2 workgroups can compute 2
```
Comment (aosewski) [inline include/ck/tensor_operation/gpu/device/device_batched_gemm_xdl.hpp:32]:
```suggestion
 * \note Using \p ComputePtrOffsetOfBatch gives us the flexibility that 2 workgroups can compute 2
```

**Review by aosewski — COMMENTED** [review]
Just found two typos.

File: include/ck/tensor_operation/gpu/device/device_batched_gemm_xdl.hpp (hunk 2 above)
Comment (j4yan) [reply include/ck/tensor_operation/gpu/device/device_batched_gemm_xdl.hpp:32]: Good catch. Thanks.

File: include/ck/tensor_operation/gpu/device/device_batched_gemm_xdl.hpp [hunk 3]
```diff
@@ -373,7 +398,7 @@ struct DeviceBatchedGemmXdl
         BGridDesc_K0_N_K1 b_grid_desc_k0_n_k1_;
         CGridDesc_M_N c_grid_desc_m_n_;
         CGridDesc_M0_N0_M1_N1_M2_M3_M4_N2 c_grid_desc_m0_n0_m1_n1_m2_m3_m4_n2_;
-        ComputeBasePtrOfStridedBatch compute_base_ptr_of_batch_;
+        ComputePtrOffsetOfStridedBatch compute_ptr_offset_of_batch;
```
Comment (asroy) [inline include/ck/tensor_operation/gpu/device/device_batched_gemm_xdl.hpp:401]: why change name?

Comment (asroy) [reply include/ck/tensor_operation/gpu/device/device_batched_gemm_xdl.hpp:401]: @j4yan Please fix the naming

**Review by asroy — APPROVED** [review]

**Pull request merged**